components/side-by-side/side-by-side.scss

```diff
@@ -0,0 +1,155 @@
+// Side by Side Component
+// Leverages design patterns from ilr-base and global styles
+
+@import '../../global/global';
+
+.cu-side-by-side {
+  display: grid;
+  gap: var(--cu-vr2);
+  align-items: center;
+  margin-bottom: var(--cu-vr2);
+
+  // Mobile-first: always stack vertically on small screens
+  grid-template-columns: 1fr;
+
+  .cu-side-by-side__media {
+    order: -1; // Image always appears first on mobile
+  }
+
+  .cu-side-by-side__content {
+    order: 1;
+  }
+
+  // Desktop: side by side layout
+  @include breakpoint(sm) {
+    grid-template-columns: 1fr 1fr;
+    gap: var(--cu-vr3);
+
+    // Reset order for desktop - variants will handle positioning
+    .cu-side-by-side__media {
+      order: initial;
+    }
+
+    .cu-side-by-side__content {
+      order: initial;
+    }
+  }
+}
+
+// Content area styling
+.cu-side-by-side__content {
+  display: flex;
+  flex-direction: column;
+  gap: var(--cu-vr1);
+}
+
+.cu-side-by-side__eyebrow {
```

Comment:
Since we're using the composite heading, I'm not sure these font style declarations do anything (the eyebrow and heading ones). I don't think they should be needed.